ruby/ql/src/utils/modeleditor/ApplicationModeEndpoints.ql

```diff
@@ -0,0 +1,12 @@
+/**
+ * @name Fetch endpoints for use in the model editor (application mode)
+ * @description A list of 3rd party endpoints (methods and attributes) used in the codebase. Excludes test and generated code.
+ * @kind table
+ * @id rb/utils/modeleditor/application-mode-endpoints
+ * @tags modeleditor endpoints application-mode
+ */
+
+// This query is empty as Application Mode is not yet supported for Ruby.
+from int n
```

Comment:
Do we really need this placeholder query? If it is necessary, perhaps make it select columns of the right names and types but still use `none()` as the where clause. That should make it more clear what should be implemented if we ever support Application Mode.

Reply:
Yes, AFAIK the model editor expects this file to be present.